### PR TITLE
[14.0][ADD] stock_picking_report_delivery_note_kit

### PR DESCRIPTION
**Pull request by KNVx**

### CODE REVIEW

File: stock_picking_report_delivery_note/models/stock_picking.py [hunk 1]
```diff
@@ -9,10 +9,14 @@
...
     _inherit = "stock.picking"
 
+    def _get_lines(self):
+        return self.move_lines
+
     def _get_amounts(self):
         self.ensure_one()
         amount_untaxed = amount_tax = 0.0
-        for line in self.move_lines:
+        self._get_lines()
```
Comment (eantones) [inline stock_picking_report_delivery_note/models/stock_picking.py]: Remove this line

File: stock_picking_report_delivery_note/views/report_stock_picking_report_delivery_note.xml [hunk 2]
```diff
@@ -243,88 +243,34 @@
...
+                            <td>
+                                <table class="sprdn-table2">
+                                    <tbody name="sprdn_document_tbody">
+
+                                        <t
+                                            t-set="grouped_moves"
+                                            t-value="o.get_moves()"
+                                        />
+                                        <t t-foreach="grouped_moves" t-as="gm">
+<!--                                            <t t-set=""-->
```
Comment (eantones) [inline stock_picking_report_delivery_note/views/report_stock_picking_report_delivery_note.xml]: Remove

File: stock_picking_report_delivery_note/views/report_stock_picking_report_delivery_note.xml [hunk 3]
```diff
@@ -243,88 +243,34 @@
...
+                                            <t
+                                                t-call="stock_picking_report_delivery_note.sprdn_document_body"
+                                                class="report_stock"
+                                            />
+                                        </t>
+                                        <!--                                        </t>-->
```
Comment (eantones) [inline stock_picking_report_delivery_note/views/report_stock_picking_report_delivery_note.xml]: Remove

File: stock_picking_report_delivery_note/views/report_stock_picking_report_delivery_note.xml [hunk 4]
```diff
@@ -243,88 +243,34 @@
...
+                                </table>
+                            </td>
+                        </tr>
                     </tbody>
                 </table>
 
                 <div class="clearfix">
                     <div id="total" class="row">
                         <t t-set="picking_amounts" t-value="o._get_amounts()" />
+
```
Comment (eantones) [inline stock_picking_report_delivery_note/views/report_stock_picking_report_delivery_note.xml]: Remove

File: stock_picking_report_delivery_note_kit/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+    "category": "Warehouse",
+    "website": "https://github.com/nuobit/odoo-addons",
+    "depends": [
+        "stock_whole_kit_constraint",
+        "stock_picking_report_delivery_note",
+    ],
+    "data": [
+        "views/report_stock_picking_report_delivery_note.xml",
+    ],
+    "installable": True,
```
Comment (eantones) [inline stock_picking_report_delivery_note_kit/__manifest__.py]: Remove

File: stock_picking_report_delivery_note_kit/__manifest__.py [hunk 6]
```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Stock picking delivery note kit",
+    "summary": """Stock picking delivery note kit""",
```
Comment (eantones) [inline stock_picking_report_delivery_note_kit/__manifest__.py]: Single quotes

File: stock_picking_report_delivery_note_kit/README.rst [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed feedback.
+
+Credits
+=======
+
+Contributors
+------------
+
+* Eric Antones <eantones@nuobit.com>
```
Comment (eantones) [inline stock_picking_report_delivery_note_kit/README.rst]: Add yourself here. In fact, create a readme folder and let the pre-commit do it for you

File: stock_picking_report_delivery_note_kit/views/report_stock_picking_report_delivery_note.xml [hunk 8]
```diff
@@ -0,0 +1,136 @@
...
+                type="text/scss"
+            />
+        </xpath>
+    </template>
+    <template
+        id="sprdn_document_kit_tbody"
+        inherit_id="stock_picking_report_delivery_note.sprdn_document"
+    >
+        <xpath expr="//tbody//t[@class='report_stock']" position="after">
+            <t t-if="gm[1]">
```
Comment (eantones) [inline stock_picking_report_delivery_note_kit/views/report_stock_picking_report_delivery_note.xml]: It should be `parent_product` use `t-set`

File: stock_picking_report_delivery_note_kit/views/report_stock_picking_report_delivery_note.xml [hunk 9]
```diff
@@ -0,0 +1,136 @@
...
+            </t>
+            <t t-else="" name="moves" />
+        </xpath>
+        <xpath expr="//t[@name='moves']" position="inside">
+            <xpath expr="//tbody//t[@class='report_stock']" position="move" />
+        </xpath>
+    </template>
+    <template id="sprdn_document_kit_body">
+        <tr>
+            <t t-set="move_lines" t-value="gm[0]" />
```
Comment (eantones) [inline stock_picking_report_delivery_note_kit/views/report_stock_picking_report_delivery_note.xml]: `t-set`

File: stock_picking_report_delivery_note_kit/views/report_stock_picking_report_delivery_note.xml [hunk 10]
```diff
@@ -0,0 +1,136 @@
...
+            <td width="15%">
+                <div t-esc="gm[1].default_code" />
```
Comment (eantones) [inline stock_picking_report_delivery_note_kit/views/report_stock_picking_report_delivery_note.xml]: `t-set` everywhere

File: stock_picking_report_delivery_note_kit/views/report_stock_picking_report_delivery_note.xml [hunk 11]
```diff
@@ -0,0 +1,136 @@
...
+                <t t-foreach="l_name.split('\n')" t-as="ln">
+                    <div t-esc="ln" />
+                </t>
+                <td width="13%" class="text-right">
+                    <!--                    <span t-field="gm[0][0].sale_line_id.qty_delivered"/>-->
+                    <!--                    <span-->
+                    <!--                        class="units"-->
+                    <!--                        t-field="gm[0][0].sale_line_id.product_uom"-->
+                    <!--                        groups="uom.group_uom"-->
+                    <!--                    />-->
```
Comment (eantones) [inline stock_picking_report_delivery_note_kit/views/report_stock_picking_report_delivery_note.xml]: Remove

**Review by eantones — CHANGES_REQUESTED** [review]

**Comment by KNVx** [comment]
@eantones , request for a squash after review, before merge

**Review by eantones — CHANGES_REQUESTED** [review]
@KNVx

**Review by eantones — APPROVED** [review]

**Pull request merged**